### PR TITLE
Re-added Undefined field to properties types

### PR DESCRIPTION
**Pull request by julioalvahash**
Re-added Undefined field to properties types.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Change Log - @itwin/insights-client
+## 0.8.1
```
Comment (Guillar1) [inline CHANGELOG.md:2]: This is a breaking change. Should be minor bump.

Comment (julioalvahash) [reply CHANGELOG.md]: Bumped to 0.9.0.

**Comment by MindaugasButkus** [comment]
@julioalvahash, @Guillar1 why was this required?

**Comment by Guillar1** [comment]
@MindaugasButkus Oof, that was a bad call. Didn't realize that it was removed in V2.

**Comment by julioalvahash** [comment]
@MindaugasButkus I tried to re-add those undefined fields to the properties types because the GM widget uses that specific field a lot but I didn't realize the 422 error detail, that was my bad, I'll rework the logic of the properties in the widget to not use that field anymore.

**Comment by MindaugasButkus** [comment]
> @MindaugasButkus I tried to re-add those undefined fields to the properties types because the GM widget uses that specific field a lot but I didn't realize the 422 error detail, that was my bad, I'll rework the logic of the properties in the widget to not use that field anymore.

It's not the fields that will cause you trouble, it's the string value "Undefined".

**Pull request merged**